scst-scan/create-supply-chain-with-app-scanning.md

```diff
@@ -0,0 +1,70 @@
+    * Supply Chain Catalog (supply-chain-catalog.apps.tanzu.vmware.com)
+    * Managed Resource Controller (managed-resource-controller.apps.tanzu.vmware.com)
+    * [Tekton](../tekton/install-tekton.hbs.md)
+  * Components:
+    * [Source](../supply-chain/reference/catalog/about.hbs.md#source-git-provider)
+    * [Buildpack](../supply-chain/reference/catalog/about.hbs.md#buildpack-build)
+    * [Trivy Scanning](../supply-chain/reference/catalog/about.hbs.md#trivy-image-scan)
+  * Scanning:
+    * [App-Scanning](./install-app-scanning.hbs.md)
+* [Tanzu Cartographer CLI Plugin](../install-tanzu-cli.hbs.md)
+
+### <a id="create-supply-chain"></a> Create Supply Chain with SCST - Scan 2.0 and Component
+
+This section covers how to create a supply chain with SCST - Scan 2.0 with either a [Trivy Supply Chain Component](./setup-supply-chain-component.md#install-trivy-supply-chain-component) or a [Customized Scanning Component](./setup-supply-chain-component.md#customize-scanning-component)
+
+#### <a id="create-supply-chain-trivy-component"></a> Create Supply Chain with SCST - Scan 2.0 and Trivy Supply Chain Component
+
+* Create a Supply Chain with App Scanning and installed [Trivy Supply Chain Component](./setup-supply-chain-component.md#install-trivy-supply-chain-component) using Tanzu Cartographer Wizard:
+```
+tanzu cartographer supply-chain wizard --name trivy-supply-chain-1.0.0 \
+--description Trivy \
+--developer-interface-group example.com \
+--developer-interface-kind TrivySC \
+--developer-interface-version v1alpha1 \
+--stages source-git-provider-1.0.0 \
+--stages buildpack-build-1.0.0 \
+--stages trivy-image-scan-0.0.1 \
+--file trivy-supply-chain.yaml
+```
+
+#### <a id="create-supply-chain-custom-component"></a> Create Supply Chain with SCST - Scan 2.0 and the custom Scanning Component
+
+* Create a Supply Chain with App Scanning and the custom Scanning Component created in the component [page](./setup-supply-chain-component.md#customize-scanning-component):
+```
+tanzu cartographer supply-chain wizard --name SCANNER-supply-chain-1.0.0 \
+--description <your scanner> \
+--developer-interface-group example.com \
+--developer-interface-kind <custom Kind workload> \
+--developer-interface-version v1alpha1 \
+--stages source-git-provider-1.0.0 \
+--stages buildpack-build-1.0.0 \
+--stages SCANNING-COMPONENT-NAME \
+--file SCANNER-supply-chain.yaml
+```
+Where:
+* SCANNING-COMPONENT-NAME is the name of the [custom scanning component](./setup-supply-chain-component.md#customize-scanning-component).
+* SCANNER is the name of the scanner from the [custom scanning component](./setup-supply-chain-component.md#customize-scanning-component).
+
+**Note:** See Tanzu Supply Chain [docs](../supply-chain/platform-engineering/how-to/supply-chain-authoring/construct-with-cli.hbs.md) for more details on how to construct a Supply Chain using the CLI.
+
+* Apply Supply Chain into the DEV-NAMESPACE where you plan to run workload
```

Comment:
**Suggestion**: This might get lost and treated as part of the custom scanning component flow. Could break it out into it's own header section to highlight it is a separate step that the Trivy flow should also follow. 
Or copy and paste into the trivy flow.